### DISCUSSION TITLE
Differences in WebUI header information

### DISCUSSION THREAD

**Discussion by fmeili1** [arendst/Tasmota#17412] [Q&A] [answered]
Hello,
I've flashed two new "Shelly Plus 1" devices and I've noticed a difference in the WebUI Main menu. The 3rd line of the header information is not available on Device 1, but on Device 2 and I wonder why and how I may change that behaviour.

Device 1: 
![pic1](https://user-images.githubusercontent.com/58776610/208128186-fc04a834-65db-45aa-b2ad-ce60fc7e2cc4.png)
![pic1A](https://user-images.githubusercontent.com/58776610/208128729-63b014ff-b8f7-43c6-af84-dac27f1d0feb.png)

Device 2:
![pic2](https://user-images.githubusercontent.com/58776610/208128405-a5146b74-0a35-4e07-9dc1-ee4f74698502.png)
![pic2A](https://user-images.githubusercontent.com/58776610/208128910-bc558513-f9e8-4b54-becc-248cf22b6397.png)

1st header line shows on both "Shelly Plus 1"
2nd header line shows the device names
**3rd header line is missing on Device 1, but shows "MI ESP32 v0921   0 / 0" on Device 2**
4th header line shows the relay status (Device 1 is ON, Device 2 is OFF)

All other configurations are the same. 

Maybe someone can answer these questions.
Why does the 3rd line for Device 1 is not showing?
What is the content of the 3rd line for Device 2 and where does it come from?
What is the meaning of "0/0" at the end of the 3rd line of Device 2?

I've noticed a difference in the Tasmota Version. Device 1 shows 12.3.1 but Device 2 shows 12.3.1.1
Even I've done the OTA firmware updates on both at the same time with the same OTA Url (http://ota.tasmota.com/tasmota32/release/tasmota32solo1.bin), they have different versions?

Thanks a lot in advance for any hints.
Frank

**Comment by sfromis** [comment, marked as answer]
The line you refer to is not a header, but an information line if you have BLE included, you likely flashed `tasmota32solo1-bluetooth` on the device where you have that line, numbers indicate how much you've paired.

The OtaUrl value does not show what you did, check the "Program Version" line on the "Information" page.

**Comment by fmeili1** [comment]
Thanks a lot for your fast answer and explenation.

You're right, I've used by mistake different images but was not aware of that. After re-flashing Device 2 via OTA with the exact same URL like for Device 1 (http://ota.tasmota.com/tasmota32/release/tasmota32solo1.bin), they both show the same information structure in the Main menu, now.

But for unknown reason, they both show still two different versions (12.3.1 vs 12.3.1.1) - but it's not really a big deal, both devices working well.

**Reply by sfromis** [reply]
The one showing 12.3.1.1 was flashed from the development binary, not the release binary. Right now, there is basically no difference, but 12.3.1 should stay the same for the next couple of months (or when next release version comes), while the development binary is replaced daily as work goes on, including occasional instances of some detail not quite working.
